libs/tdb/files/tdb.cache.txt

```diff
@@ -0,0 +1,36 @@
+Checking simple C program: OK
```

Comment:
By the way, what happens if anything in the toolchain changes and these values are no longer correct?